efel/tests/testdata/allfeatures/expectedresults.json

```diff
@@ -377,15 +377,10 @@
     "amp_drop_second_last": [
         4.787221503925529
     ],
-    "burst_ISI_indices": [
-        3
-    ],
-    "burst_mean_freq": [
-        3.9840637450221186,
-        7.990411506199836
-    ],
+    "burst_ISI_indices": [],
+    "burst_mean_freq": [],
     "burst_number": [
-        2
```

Comment:
Why did this change? I thought the implementation of burst_number was not changed?

Reply:
The implementation of burst_number was not changed. This is because I changed the default value of burst_factor. Some peaks are considered to be part of a burst when `burst_factor` is `1.5`, but they are no longer considered part of a burst when `burst_factor` is `2`.

Reply:
Do you mind plotting the trace for which this changed, so that we can have an idea of the effect.

Reply:
Sure, here it is:

![testdata](https://user-images.githubusercontent.com/72930209/192549816-0421a4f8-47e0-48a0-a9f3-03e723d5799e.png)

here is the list of all the ISI values:
`[203.3 494.7 306.  675.5 250.3]`
With the old implementation:
1st ISI is skipped.
median of 2nd and 3rd ISI: `400.35`
When `burst_factor` is `1.5`: 4th ISI is larger than `600` (`400.35 * 1.5`), and the following ISI is smaller than `675/1.5`, so a new burst (consisting of the two last spikes) is detected
When `burst_factor = 2`: 4th ISI is smaller than `800.7 `(`400.35 * 2`), so no new burst is detected.

By the way, should we consider these to be bursts?